modules/my-module/android/src/main/java/expo/modules/mymodule/MyModule.kt

```diff
@@ -32,8 +38,19 @@ class MyModule : Module() {
 
     // Defines a JavaScript function that always returns a Promise and whose native code
     // is by default dispatched on the different thread than the JavaScript runtime runs on.
+    // TODO: this is the other side of the bridge!
```

Comment:
Not about this comment which are totally cool to leave LOL but closest line to what I want to say which is we should probably have this as a new function or rename the function